engine/class_modules/sc_death_knight.cpp

```diff
@@ -9427,6 +9535,9 @@ void death_knight_t::init_gains()
   // Unholy
   gains.apocalypse                       = get_gain( "Apocalypse" );
   gains.festering_wound                  = get_gain( "Festering Wound" );
+
+  // Shadowlands / Covenants
+  gains.swarming_mist                    = get_gain( "swarming_mist" );
```

Comment:
You don't need to tokenize the name here
```suggestion
  gains.swarming_mist                    = get_gain( "Swarming Mist" );
```